File: client/src/app/shared/models/tarif.ts
export interface IServiciu {
        id: number;
        cod: string;
        den: string;
        um: string;
}

export interface IServiciuToSaveDto {
        cod: string;
        den: string;
        um: string;
}

export interface ITarifToSaveDto {
        clientId: number;
        serviciuId: number;
        pret: number;
        coefKm: number;
        procDisc: number;
}

export interface ITarif {
        id: number;
        clientId: number;
        serviciuId: number;
        cod: string;
        den: string;
        um: string;
        pret: number;
        coefKm: number;
        procDisc: number;
}

File: Core/Entities/Tarif.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Entities 
{
    public class Tarif : BaseEntity
    {
        public int ClientId { get; set; }
        public Client Client { get; set; }

        public int ServiciuId { get; set; }
        public Serviciu Serviciu { get; set; }

        public decimal? Pret { get; set; }
        public byte? CoefKm { get; set; }
        public byte? ProcDisc { get; set; }

    }
}
